resolwe/flow/serializers/fields.py

```diff
@@ -11,63 +11,65 @@
 from .descriptor import DescriptorSchemaSerializer
 
 
-class ResolweSlugRelatedField(RelatedField):
+class DictRelatedField(RelatedField):
```

Comment:
Should we call this `ResolweRelatedField`? That would be a dict field that can contain either id or slug, and can be hydrated or not.